package.json

```diff
@@ -53,32 +71,37 @@
   },
   "devDependencies": {
     "@lavamoat/allow-scripts": "^2.3.1",
+    "@lavamoat/preinstall-always-fail": "^1.0.0",
     "@metamask/auto-changelog": "^3.1.0",
-    "@metamask/eslint-config": "^11.1.0",
-    "@metamask/eslint-config-jest": "^11.1.0",
-    "@metamask/eslint-config-nodejs": "^11.1.0",
-    "@metamask/eslint-config-typescript": "^11.1.0",
-    "@types/jest": "^27.0.6",
-    "@types/node": "^16.18.50",
-    "@typescript-eslint/eslint-plugin": "^5.59.1",
+    "@metamask/eslint-config": "^12.0.0",
+    "@metamask/eslint-config-jest": "^12.0.0",
+    "@metamask/eslint-config-nodejs": "^12.0.0",
+    "@metamask/eslint-config-typescript": "^12.0.0",
+    "@types/jest": "^28.1.6",
+    "@types/node": "^16",
```

Comment:
This matches the module template: https://github.com/MetaMask/metamask-module-template/blob/085b701d9084b45179caef9db1c0b60afa659a78/package.json#L52